docs/3rdpartylicenses.txt

```diff
@@ -0,0 +1,211 @@
+core-js@2.5.5
```

Comment:
are you sure you want to commit this file?

Reply:
Oh my bad I am not even sure how it was added to my commit probably generated with "ng build" command.